package.json

```diff
@@ -80,7 +80,13 @@
             "plugin:@taiga-ui/experience/entry-points"
         ],
         "rules": {
-            "jest/prefer-importing-jest-globals": "off"
+            "jest/prefer-importing-jest-globals": "off",
+            "prettier/prettier": [
+                "error",
+                {
+                    "endOfLine": "auto"
+                }
+            ]
```

Comment:
Should it be reverted ? 🤔

Reply:
I added this line to taiga linters, it will be updated later